canvasapi/quiz.py

```diff
@@ -16,6 +16,32 @@ class Quiz(CanvasObject):
     def __str__(self):
         return "{} ({})".format(self.title, self.id)
 
+    def broadcast_message(self, conversations, **kwargs):
+        """
+        Send a message to unsubmitted or submitted users for the quiz.
+
+        :calls: `POST /api/v1/courses/:course_id/quizzes/:id/submission_users/message \
+        <https://canvas.instructure.com/doc/api/quiz_submission_user_list.html#method.quizzes/quiz_submission_users.message>`_
+
+        :param conversations: The ID of the question group.
+        :type conversations: :class:`canvasapi.quiz.QuizUserConversation`
```

Comment:
What is `QuizUserConversation`? I don't see it anywhere.

Also, if this is supposed to allow an ID, we should be leveraging `obj_or_id`